app/routes/user.py

```diff
@@ -3,4 +3,22 @@
 """
 from fastapi import APIRouter
 
-userRouter: APIRouter = APIRouter()
+from app.models.pydantic.UserModel import PydanticUserModify
+
+import app.services.UserService as UserService
+
+userRouter: APIRouter = APIRouter(prefix="/user")
+
+@userRouter.patch("/{user_id}", status_code=205)
+async def modify_user(user_id: int, user_model: PydanticUserModify) -> None:
+    """
+    This controllers is used when modifying user informations.
+    """
+    await UserService.modify_user(user_id, user_model)
+
+@userRouter.delete("/{user_id}", status_code=204)
```

Comment:
Wrong error code used. Should be 205.
https://developer.mozilla.org/fr/docs/Web/HTTP/Status/205

Reply:
Done

Reply:
You can click the "resolve conversation" button when the subject is closed.